packages/grafana-data/src/geo/layer.ts

```diff
@@ -58,6 +58,9 @@ export interface MapLayerOptions<TConfig = any> {
   // Common method to define geometry fields
   location?: FrameGeometrySource;
 
+  // Defines which data query is associated with the layer
+  dataquery?: string;
```

Comment:
```suggestion
  filterData?: MatcherConfig;
```
and refId can be one of the options

Reply:
I'm not sure what I'm supposed to do here. this field corresponds to the builder path specified [here](https://github.com/grafana/grafana/blob/e1dc2b028f54d1b553fa837030ec8c3321a9d67e/public/app/plugins/panel/geomap/editor/layerEditor.tsx#L88). I haven't been able to find other examples that use Matchers in this way.